bears/jinja2/Jinja2Bear.py

```diff
@@ -345,6 +394,9 @@ def run(self,
         :param variable_spacing:
             The number of spaces a variable block should be spaced with.
             Default is 1.
+        :param statement_spacing:
+            The number of spaces a statement block should be spaced with.
+            Default is 1.
```

Comment:
hm it's actually redundant to mention the default value, coala displays them automatically when querying bear infos. But due to consistency you can keep it. We file an issue after this PR to remove this.

Reply:
Will keep in mind to open an issue for this once this PR approved. :)